swift-sdk/Internal/InternalIterableAPI.swift

```diff
@@ -135,8 +122,7 @@ final class InternalIterableAPI: NSObject, PushTrackerProtocol, AuthProvider {
         
         storeIdentifierData()
         
-        requestNewAuthToken()
-        loginNewUser()
+        onLogin()
```

Comment:
So now we have `onLogin` which calls `loginNewUser`. I think it is a little confusing because `onLogin` should be called **after** login is completed. 

I think `loginNewUser` should be called `onLogin` and `onLogin` should be called `loginNewUser`.

Or maybe `loginNewUser` should be renamed to updateBackendOnLogin.

Reply:
Hmm, yeah, I agree. I'll find a new name for `loginNewUser`, I think that would be the best pick for a name revision.